### PR TITLE
podmanv2 info

### PR DESCRIPTION
**Pull request by baude**
add ability to run info for v2

Signed-off-by: Brent Baude <bbaude@redhat.com>

### CODE REVIEW

File: cmd/podmanV2/system/info.go [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+	"github.com/containers/libpod/cmd/podmanV2/registry"
+	"github.com/containers/libpod/pkg/domain/entities"
+	"github.com/spf13/cobra"
+	"gopkg.in/yaml.v2"
+)
+
+var (
+	infoDescription = `Display information pertaining to the host, current storage stats, and build of podman.
+
+  Useful for the user and when reporting issues.
```
Comment (mheon) [inline cmd/podmanV2/system/info.go:18]: Useful for the user how?

**Comment by mheon** [comment]
One nit otherwise LGTM

File: cmd/podmanV2/system/info.go (hunk 1 above)
Comment (baude) [reply cmd/podmanV2/system/info.go:18]: straight copy from the old client ...

**Pull request merged**